ProteoformSuiteGUI/ExperimentTheoreticalComparison.cs

```diff
@@ -385,6 +385,7 @@ private void nUD_PeakCountMinThreshold_ValueChanged(object sender, EventArgs e)
                 Parallel.ForEach(SaveState.lollipop.et_peaks, p =>
                 {
                     p.Accepted = p.peak_relation_group_count >= SaveState.lollipop.min_peak_count_et;
+                    p.count_nearby_decoys(SaveState.lollipop.ed_relations.Values.SelectMany(v => v).ToList());
```

Comment:
Why not adapt the peak_acceptibility changed method to change those accepted booleans for decoy relations, rather than recounting? That counting step is pretty computationally expensive.